main.py

```diff
@@ -110,15 +110,17 @@ def markers():
     severe = int(request.values['show_severe'])
     light = int(request.values['show_light'])
     inaccurate = int(request.values['show_inaccurate'])
+    mark_on = bool(request.values['mark_on'])
```

Comment:
Generally it's better to use complete words, since clarity is more important than saving bits. Also, why `_on` and not the former convention of `show_`?